src/libgit2/odb_mempack.c

```diff
@@ -209,3 +209,14 @@ int git_mempack_new(git_odb_backend **out)
 	*out = (git_odb_backend *)db;
 	return 0;
 }
+
+int git_mempack_empty(git_odb_backend *_backend)
+{
+	struct memory_packer_db *db = (struct memory_packer_db *)_backend;
+	GIT_ASSERT_ARG(_backend);
+
+	if (git_odb_mempack_oidmap_size(&db->objects) || db->commits.size)
```

Comment:
I'm curious about the logical or here - is there ever a case where these two would disagree?

Reply:
Done, no need to check number of commits, thank you!